client.go

```diff
@@ -326,6 +350,7 @@ func (c *Client) DeleteByID(ctx context.Context, pinID string) error {
 	return nil
 }
 
+// TODO: Remove? (not sure what is the purpose of this - it just adds pin, does not modify nro replace anything)
 func (c *Client) Modify(ctx context.Context, pinID string, cid cid.Cid, opts ...AddOption) (PinStatusGetter, error) {
```

Comment:
Isn't `PinsRequestidPost` the replace function? https://ipfs.github.io/pinning-services-api-spec/#tag/pins/paths/~1pins~1{requestid}/post

Reply:
Indeed, I think it was late and the name confused me.
I've removed todo and renamed `Modify` to `Replace` to match language used in Pinning Service API spec.